api/src/main/java/ca/bc/gov/educ/api/gradstudent/service/GraduationStatusService.java

```diff
@@ -878,7 +875,6 @@ public void deleteStudentOptionalProgram(UUID studentID, UUID optionalProgramID,
         // Validation
         GraduationStudentRecord graduationStudentRecord = getGraduationStatus(studentID);
         validateStudent(graduationStudentRecord);
-        validateOptionalProgram(optionalProgramID, accessToken);
```

Comment:
Just confirm this is intention to remove the validation here.   Thanks.

Reply:
Yes as the intention of the method is to delete the optional program, I felt that the additional validation was not needed.